Makefile

```diff
@@ -9,17 +9,25 @@ PREFIX ?= ${ROOT}/usr
 INSTALL_DIR ?= ${PREFIX}
 MODULE_DIR := ${ROOT}/modules
 
-DEB_COMPONENT := clearwater-snmp-handlers
-DEB_MAJOR_VERSION := 1.0${DEB_VERSION_QUALIFIER}
+PKG_COMPONENT := clearwater-snmp-handlers
+PKG_MAJOR_VERSION ?= 1.0
+
+# Define DEB_NAMES and RPM_NAMES separately as we don't have RPMs for all the
+# components yet.
 DEB_NAMES := clearwater-snmp-handler-cdiv clearwater-snmp-alarm-agent clearwater-snmp-handler-memento-as clearwater-snmp-handler-memento clearwater-snmp-handler-astaire clearwater-snmp-alarm-agent-dbg
+RPM_NAMES := clearwater-snmp-alarm-agent clearwater-snmp-alarm-agent-debuginfo
```

Comment:
As discussed, I think it would be cleaner to

*   remove `-dbg` and `-debuginfo` from `PKG_NAMES`, `DEB_NAMES` and `RPM_NAMES`
*   in cw-rpm.mk,

    *    build the spec files for all `RPM_NAMES`
    *    copy all packages named by `RPM_NAMES` or named by `RPM_NAMES` with a `-debuginfo` suffix

*   in cw-deb.mk, copy all packages named by `DEB_NAMES` or named by `DEB_NAMES` with a `-dbg` suffix
*   remove `RPM_SPECS` entirely (as it's not required).

Sound reasonable?